controllers/chat/getChannelDetail.js

```diff
@@ -37,7 +37,7 @@ const getChannelDetail = async (req, res) => {
   }
 
   const {betterChannelMember, betterChannelMemberObject} =
-    await BetterSocialCore.chat.updateBetterChannelMembers(channel, createdChannel);
+    await BetterSocialCore.chat.updateBetterChannelMembers(channel, createdChannel, true);
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The `updateBetterChannelMembers` function now has a third argument set to `true`. Without context or comments, it's unclear what this boolean value represents. It would be better to use a well-named variable instead of a direct boolean value to improve code readability and maintainability.
```diff
- await BetterSocialCore.chat.updateBetterChannelMembers(channel, createdChannel, true);
+ const updateAllMembers = true;
+ await BetterSocialCore.chat.updateBetterChannelMembers(channel, createdChannel, updateAllMembers);
```
This change assumes that the third parameter is used to indicate whether all members should be updated. Please replace `updateAllMembers` with an appropriate name based on your actual implementation.


<!-- This is an auto-generated comment by OSS CodeRabbit -->